wallet/build.gradle

```diff
@@ -216,6 +216,9 @@ android {
     }
 
     buildTypes {
+        release {
+            signingConfig signingConfigs.release
+        }
```

Comment:
This needs to be removed as it breaks GitHub Actions.

Reply:
fixed